packages/web/modals/validator-squad.tsx

```diff
@@ -78,7 +93,21 @@ const ValidatorSquadContent: FunctionComponent<ValidatorSquadContentProps> =
       return delegationsMap;
     }, [userValidatorDelegations]);
 
-    const data: Validator[] = useMemo(
+    // table
+    const [sorting, setSorting] = useState<SortingState>([
+      { id: "myStake", desc: true },
+    ]);
+    const columnHelper = createColumnHelper<Validator>();
+
+    // search
+    const [searchTerm, setSearchTerm] = useState("");
```

Comment:
Note: we have a hook `useFilteredData` that exposes a simple API for doing fuzzy & cached search via Fuse.js